app/build.gradle

```diff
@@ -63,13 +63,15 @@ android {
 }
 
```

Comment:
ditto id 'org.jetbrains.kotlin.plugin.serialization' version '1.8.10'

Reply:
There is no serialization plugin in this branch